keyboards/crkbd/keymaps/nimishgautam/rules.mk

```diff
@@ -0,0 +1,21 @@
+MOUSEKEY_ENABLE = yes    # Mouse keys
```

Comment:
You only need the options that are different from the default here. Anything else should be removed.

Reply:
K, I went through the parent config file and removed the ones that were redundant between them